src/crossover.jl

```diff
@@ -62,8 +62,8 @@ of the chromosome is randomly selected from one of the parents.
 function cross(::UniformCrossover, a, b; rng=Random.GLOBAL_RNG)
     child = copy(a)
 
-    @inbounds for i in eachindex(a)
-        child[i] = rand(rng) < 0.5 ? b[i] : continue
+    for i = eachindex(a)
```

Comment:
```suggestion
    for i in eachindex(a)
```
This is just part of Invenia's blue naming guidelines we follow